src/GameLogic.cpp

```diff
@@ -21,30 +22,43 @@ PlayerActor GameLogic::getPlayer(){
   return player;
 }
 
+void GameLogic::EPressed(){
```

Comment:
problem is, the player can press E at any time, even if the player is not standing next to the item. So the etracker will always increment